website/client/router.js

```diff
@@ -101,6 +103,14 @@ export default new VueRouter({
             },
           ],
         },
+        { name: 'challenges', path: 'challenges', component: Page },
```

Comment:
why challenges under user? also it's a duplicate, there's another challenge entry a few lines above